chapter_15_advanced_forms.asciidoc

```diff
@@ -1117,14 +1121,9 @@ real app.
 We have one unit test file for each of our key source code files.  Here's
 a recap of the biggest (and highest-level) one, 'test_views' (the listing
 shows just the key tests and assertions, and your order may vary):
-// CSANAD: Why are we only showing ListViewTest? HomePageTest, NewListTest
 
-// RITA: Why is this in a box? I don't think it needs to be because it's a code block that is discussed in the preceding body text. Did you do this so it would double as the "wrap-up" box that you have at the end of most chapters? It looks a bit strange this way. 
```

Comment:
i actually _did_ do it so it would be the wrap_up box.

maybe we can prefix the title with "Wrap-up: What to Test in Views" instead?